source/includes/quick-start/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	uri = os.Getenv("MONGODB_URI")
```

Comment:
Minor, but this is inconsistent with how the URI is passed in the findOne example, which is directly included in the source:
https://github.com/mongodb/docs-golang/pull/2/files#diff-9080c13327b8e149406ba2a276b31c02c0e664eb9db2a5a6c8c348f2a9a037ffR13

I can see an argument for either way. Passing through an environment variable is safer, and will probably save some new users from accidentally committing credentials. But for a quick getting started guide, it is one less step if the uri is directly in the source. Either way, it may be good to be consistent.

If you decide to stick with an environment variable, I'd suggest adding a check in `main` and panicking if it is unset. Currently, if you forget to set the environment variable `MONGODB_URI`, you'll get the following error, which might be unclear to users:

```
$ go run main.go
2021/05/07 11:58:56 error parsing uri: scheme must be "mongodb" or "mongodb+srv"
panic: error parsing uri: scheme must be "mongodb" or "mongodb+srv"
```

Reply:
I had thought about this but initially dismissed it as perhaps being too detailed. I'll run my idea by you in Slack.